src/measure_theory/bochner_integration.lean

```diff
@@ -1042,67 +1042,76 @@ open continuous_linear_map measure_theory.simple_func
 
-lemma integral_non_integrable (h : ¬ integrable f) : integral f = 0 :=
+lemma integral_non_integrable (h : ¬ integrable f) : (∫ a, f a) = 0 :=
 integral_undef $ not_and_of_not_right _ h
 
-lemma integral_non_measurable (h : ¬ measurable f) : integral f = 0 :=
+lemma integral_non_measurable (h : ¬ measurable f) : (∫ a, f a) = 0 :=
 integral_undef $ not_and_of_not_left _ h
 
 variables (α β)
-@[simp] lemma integral_zero : integral (0 : α → β) = 0 :=
+@[simp] lemma integral_zero : (∫ a:α, (0:β)) = 0 :=
 by rw [integral_eq, l1.of_fun_zero, l1.integral_zero]
 
 variables {α β}
 
 lemma integral_add
   (hfm : measurable f) (hfi : integrable f) (hgm : measurable g) (hgi : integrable g) :
-  integral (f + g) = integral f + integral g :=
+  (∫ a, f a + g a) = (∫ a, f a) + (∫ a, g a) :=
 by rw [integral_eq, integral_eq f hfm hfi, integral_eq g hgm hgi, l1.of_fun_add, l1.integral_add]
 
-lemma integral_neg (f : α → β) : integral (-f) = - integral f :=
+lemma integral_neg (f : α → β) : (∫ a, -f a) = - (∫ a, f a) :=
 begin
   by_cases hf : measurable f ∧ integrable f,
-  { rw [integral_eq f hf.1 hf.2, integral_eq (- f) hf.1.neg hf.2.neg, l1.of_fun_neg,
+  { rw [integral_eq f hf.1 hf.2, integral_eq (λa, - f a) hf.1.neg hf.2.neg, l1.of_fun_neg,
     l1.integral_neg] },
-  { have hf' : ¬(measurable (-f) ∧ integrable (-f)),
+  { have hf' : ¬(measurable (λa, -f a) ∧ integrable (λa, -f a)),
     { rwa [measurable_neg_iff, integrable_neg_iff] },
     rw [integral_undef hf, integral_undef hf', neg_zero] }
 end
 
 lemma integral_sub
   (hfm : measurable f) (hfi : integrable f) (hgm : measurable g) (hgi : integrable g) :
-  integral (f - g) = integral f - integral g :=
+  (∫ a, f a - g a) = (∫ a, f a) - (∫ a, g a) :=
 by simp only [sub_eq_add_neg, integral_neg, integral_add, measurable_neg_iff, integrable_neg_iff, *]
 
-lemma integral_smul (r : ℝ) (f : α → β) : integral (r • f) = r • integral f :=
+lemma integral_smul (r : ℝ) (f : α → β) : (∫ a, r • (f a)) = r • (∫ a, f a) :=
 begin
   by_cases hf : measurable f ∧ integrable f,
-  { rw [integral_eq f hf.1 hf.2, integral_eq (r • f), l1.of_fun_smul, l1.integral_smul] },
+  { rw [integral_eq f hf.1 hf.2, integral_eq (λa, r • (f a)), l1.of_fun_smul, l1.integral_smul] },
   { by_cases hr : r = 0,
     { simp only [hr, measure_theory.integral_zero, zero_smul] },
-    have hf' : ¬(measurable (r • f) ∧ integrable (r • f)),
+    have hf' : ¬(measurable (λa, r • f a) ∧ integrable (λa, r • f a)),
     { rwa [← measurable_smul_iff hr f, ← integrable_smul_iff hr f] at hf },
     rw [integral_undef hf, integral_undef hf', smul_zero] }
 end
 
+lemma integral_mul_left (r : ℝ) (f : α → ℝ) : (∫ a, r * (f a)) = r * (∫ a, f a) :=
+integral_smul r f
+
+lemma integral_mul_right (r : ℝ) (f : α → ℝ) : (∫ a, (f a) * r) = (∫ a, f a) * r :=
+by { simp only [mul_comm], exact integral_mul_left r f }
+
+lemma integral_on_div (r : ℝ) (f : α → ℝ) : (∫ a, (f a) / r) = (∫ a, f a) / r :=
+by { simp only [div_eq_mul_inv], apply integral_mul_right }
```

Comment:
You can use `integral_mul_right r⁻¹ f` directly and let Lean unfold a definition for you, or leave as is if you prefer a more explicit style.